test/system/InfluxDBST.cxx

```diff
@@ -251,12 +252,89 @@ namespace influxdb::test
             const Point::FieldSet& fields{point.getFieldSet()};
             // String fields are put in the tags (see above)
             CHECK(fields.size() == 0);
+            db->execute("drop database " + dbName);
+        }
+    }
+
+    TEST_CASE("UDP Transport", "[InfluxDBST]")
```

Comment:
Since this is some kind of new test suite, maybe be should extract this to a dedicated source file (to keep each on it's own topic).